brave/src/test/java/com/linecorp/armeria/server/brave/ArmeriaHttpServerAdapterTest.java

```diff
@@ -151,26 +151,34 @@ public void parseClientIpAndPort() throws Exception {
     }
 
     @Test
-    public void route() {
+    void route() {
         final ServiceRequestContext context = mock(ServiceRequestContext.class);
         when(requestLog.context()).thenReturn(context);
         when(context.route()).thenReturn(Route.builder().path("/foo/:bar/hoge").build());
         assertThat(ArmeriaHttpServerAdapter.get().route(requestLog)).isEqualTo("/foo/:/hoge");
     }
 
     @Test
-    public void route_prefix() {
+    void route_prefix() {
         final ServiceRequestContext context = mock(ServiceRequestContext.class);
         when(requestLog.context()).thenReturn(context);
         when(context.route()).thenReturn(Route.builder().path("exact:/foo").build());
         assertThat(ArmeriaHttpServerAdapter.get().route(requestLog)).isEqualTo("/foo");
     }
 
     @Test
-    public void route_pathWithPrefix() {
+    void route_pathWithPrefix_glob() {
         final ServiceRequestContext context = mock(ServiceRequestContext.class);
         when(requestLog.context()).thenReturn(context);
         when(context.route()).thenReturn(Route.builder().pathWithPrefix("/foo/", "glob:bar").build());
-        assertThat(ArmeriaHttpServerAdapter.get().route(requestLog)).isEqualTo("/foo/ ^/(?:.+/)?bar$");
+        assertThat(ArmeriaHttpServerAdapter.get().route(requestLog)).isEqualTo("/foo/**/bar");
```

Comment:
@adriancole You will like this, right? :wink:

Reply:
no..  I love it :)